mount/mount.c

```diff
@@ -54,6 +58,26 @@ static int dns_lookup(const char *hostname, struct sockaddr_in *addr)
 	return 0;
 }
 
+static int fsopen(const char *fsname, unsigned int flags) {
```

Comment:
this should be long.

Reply:
Why? They all seem to be ints here:

https://github.com/torvalds/linux/blob/60e720931556fc1034d0981460164dcf02697679/samples/vfs/test-fsmount.c#L72